doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,44 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt when creating projects in level 3,4,5?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can override the existing default values in the prompt by creating and editing ``~/.skpkgrc``. For example, to change some default values prompted when creating a new package in level 5, please follow the quick steps below:
+
+1. Create ``~/.skpkgrc`` by running ``touch ~/.skpkgrc``.
+2. Copy and paste the following snippets.
+
+.. code-block:: json
```

Comment:
here I think we could make it clearer how the user changes the default values (by typing over sanjoon's name".  We could also think about replacing them with words in `<angle-brackets>` like this and in hte instructions say "replace words in angle brackets with values relevant for your local installation or sthg.

Please also mention that any that you simply delete any lines for prompts that you don't want to modify from the package defaults.  btw, have we tested that this works?